newswires/app/conf/SearchPresets.scala

```diff
@@ -2,53 +2,167 @@ package conf
+
+      > ; Category Codes
+      >  2	JC=a*				>w4apapi#NY:for
+      >  2	JC=d*				>w4apapi#NY:fea
+      >  2	JC=e*				>w4apapi#NY:fea
+      >  2	JC=f*				>w4apapi#NY:fin
+      >  2	JC=i*				>w4apapi#NY:for
+      >  2	JC=s*				>w4apapi#NY:spt
+      >  2	JC=t*				>w4apapi#NY:fea
+      >  2	JC=w*				>w4apapi#NY:for
+      > ; Default
+      >  2	JC=*				>w4apapi#NY:for
+
+    * The fingerpost system runs top to bottom, and '>' tells it to stop once it finds a match, so an item with
+    * category code 'JC:ae' would be bucketed as 'NY:for' and not 'NY:fea', and an item with category code 'JC:ew'
+    * would be bucketed as 'NY:fea' rather than 'NY:for'.
+    * We're inclined to exclude sports, entertainment, finance, and technology news from this preset instead, even
+    * if they have e.g. code 'a' (US news) code, because they're likely to be less relevant to International desk.
+    * However, we should remain open to changing this in response to user feedback.
+    */
+  // format: on
+  private val ApWorld = List(
+    SearchParams(
+      text = None,
+      suppliersIncl = List("AP"),
+      keywordIncl = List("World news"),
+      categoryCodesIncl = List("apCat:i", "apCat:a", "apCat:w"),
+      categoryCodesExcl = List("apCat:s", "apCat:e", "apCat:f")
+    )
+  )
+
+  private val ReutersWorld = List(
+    SearchParams(
+      text = None,
+      suppliersIncl = List("REUTERS"),
+      categoryCodesIncl = List(
+        "REUTERS:RWS",
+        "REUTERS:RWSA"
+      ),
+      categoryCodesExcl = List(
+        "MCC:SPO"
+      )
+    ),
+    SearchParams(
+      text = None,
+      suppliersIncl = List("REUTERS"),
+      categoryCodesIncl = List(
+        "MCC:OVR",
+        "MCC:QFE",
+        "MCCL:OVR",
+        "MCCL:OSM",
+        "N2:US"
+      ),
+      categoryCodesExcl = List(
+        "MCC:DED",
+        "MCC:SPO",
+        "MCC:OEC",
+        "N2:GB",
+        "N2:COM",
+        "N2:ECI"
+      )
+    ),
+    SearchParams(
+      text = Some("News Summary"),
+      suppliersIncl = List("REUTERS"),
+      categoryCodesIncl = List(
+        "MCC:OEC"
+      ),
+      categoryCodesExcl = List(
+        "N2:GB",
+        "N2:COM",
+        "N2:ECI"
+      )
+    )
+  )
+
+  private val AapWorld = List(
+    SearchParams(
+      text = None,
+      suppliersIncl = List("AAP"),
+      keywordExcl = List("Sports"),
+      categoryCodesExcl = Categories.sportsRelatedNewsCodes
+    )
+  )
+
+  private val AfpWorld = List(
+    SearchParams(
+      text = None,
+      suppliersIncl = List("AFP"),
+      categoryCodesExcl = List("afpCat:SPO")
+    )
+  )
+
+  private val MinorAgenciesWorld = List(
+    SearchParams(
+      text = None,
+      suppliersIncl = List("MINOR_AGENCIES"),
+      categoryCodesExcl = List("N2:GB")
+    )
+  )
+
+  private val PaHome = List(
+    SearchParams(
+      text = None,
+      suppliersIncl = List("PA"),
+      categoryCodesIncl = List(
```

Comment:
the results from this set of params look pretty good to me, but I'm not sure where the list of category codes came from originally -- have you been able to find documentation of the PA catcodes somewhere?